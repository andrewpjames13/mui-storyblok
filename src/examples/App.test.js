import React from 'react';
import { shallow } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import App from './App';


describe('<MuiAppBar />', () => {
  it('renders MuiAppBar', () => {
    const comp = shallow(<App />);
    expect(comp).toBeDefined();
  });

  test('snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <App />
      </MemoryRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
