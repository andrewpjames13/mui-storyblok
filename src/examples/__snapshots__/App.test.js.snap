// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MuiAppBar /> snapshot 1`] = `
<index.mjs
  accessToken="9K1dqG8gQkc1jeVbOWRDUAtt"
  theme={Object {}}
  version="draft"
/>
`;
